sdk/ai/azure-ai-projects/azure/ai/projects/models/_enums.py

```diff
@@ -303,6 +303,19 @@ class MessageStreamEvent(str, Enum, metaclass=CaseInsensitiveEnumMeta):
     """Event sent before a message is completed. The data of this event is of type ThreadMessage"""
 
 
+class OpenApiAuthType(str, Enum, metaclass=CaseInsensitiveEnumMeta):
```

Comment:
OpenApi? strange. Or this supposed to be OpenAI?

Reply:
This auth type (connection/anonymous/managed_identity) is currently specific to the openapi tool schema.